digdag-profiler/src/main/java/io/digdag/profiler/ConfigElementLoader.java

```diff
@@ -0,0 +1,62 @@
+package io.digdag.profiler;
+
+import com.google.common.collect.ImmutableMap;
+import io.digdag.cli.Command;
+import io.digdag.cli.SystemExitException;
+import io.digdag.client.config.ConfigElement;
+import io.digdag.core.config.PropertyUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Properties;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+
+public class ConfigElementLoader
```

Comment:
Not strong opinion, but I guess this class can be available not only profiler but also other module.
So move to digdag-client may be better.

Reply:
ConfigElementLoader was created only to call `io.digdag.cli.Command#loadSystemProperties` and 
we'll move digdag-profiler to digdag-cli as `Profile` class which inherits `Command` class, so we don't need this class anymore.